openeo/rest/auth/config.py

```diff
@@ -18,22 +18,29 @@
 from openeo.config import get_user_config_dir, get_user_data_dir
 from openeo.util import rfc3339, deep_get, deep_set
 
+if platform.system() == 'Windows':
```

Comment:
Maybe instead of an "if windows: import"  just do "try: import ...; except: oschmod=None"
then the openeo client package is even usable in a windows context where  oschmod can not be installed for some reason

Reply:
(e.g. it would allow a test run on Linux to test the "windows code path")

Reply:
Good idea. I'm on it.

Reply:
Done.